tftrt/examples/image-classification/image_classification.py

```diff
@@ -609,6 +613,7 @@ def print_dict(input_dict, str='', scale=None):
     print_dict(num_nodes, str='num_nodes')
     print_dict(graph_sizes, str='graph_size(MB)', scale=1./(1<<20))
     print_dict(times, str='time(s)')
+    print("url: " + get_netdef(args.model).get_url())
```

Comment:
Could you move this to before printing `print_dict(num_nodes, str='num_nodes')` and after `print_dict(vars(args))`

Reply:
done